ticket_metrics.view.lkml

```diff
@@ -306,6 +306,14 @@ view: ticket_metrics {
     sql: ${TABLE}.reply_time_in_minutes__calendar / 60 ;;
   }
 
+  dimension: coalesce_reply_time_and_first_resolution_time_in_minutes__calendar {
```

Comment:
Not the end of the world, but really easier if it read: `reply_time_coalesced_with_first_resolution_time__in_minutes__calendar`

Reply:
changed